Userland/Libraries/LibGfx/ICC/Profile.cpp

```diff
@@ -558,7 +558,9 @@ ErrorOr<void> Profile::read_header(ReadonlyBytes bytes)
 
 ErrorOr<NonnullRefPtr<TagData>> Profile::read_tag(ReadonlyBytes bytes, u32 offset_to_beginning_of_tag_data_element, u32 size_of_tag_data_element)
 {
-    // FIXME: Check 4-byte alignment on incoming offset.
```

Comment:
Seems silly to have a previous commit adding this FIXME in the same PR, please drop that.

Reply:
Done. (Originally I thought that commit would be in a separate PR, but then I had a bonus 5 minutes.)

Reply:
I think that rebase didn't quite work out - `LibGfx: Add a FIXME to ICC::Profile` is still there, with the removal being in the following commit :)

Reply:
What about now?